02.1/api/validators/user.validators.ts

```diff
@@ -0,0 +1,48 @@
+import * as Joi from '@hapi/joi';
+
+import {
+  ContainerTypes,
+  // Extend from this to define a valid schema type/interface
+  ValidatedRequestSchema,
+  // Creates a validator that generates middlewares
+  createValidator,
+} from 'express-joi-validation';
+
+const validator = createValidator({
+  passError: true,
+});
+
+const createUserBodySchema = Joi.object({
+  login: Joi.string().required(),
+  password: Joi.string().alphanum().required(),
+  age: Joi.number().min(4).max(130).required(),
```

Comment:
It might be a validation library issue, but service accepts age as a string, for example:
```
 {
    "login": "myUser1",
    "password": "8888",
    "age": "55"
}
```